lib/berachain-light-client-types/src/client_state.rs

```diff
@@ -1,12 +1,11 @@
-use unionlabs::{hash::H160, uint::U256};
+use unionlabs::{hash::H160, id::ClientId, uint::U256};
 
 #[derive(Debug, Clone, PartialEq)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 pub struct ClientState {
-    pub comet: tendermint_light_client_types::ClientState,
-    pub execution_chain_id: U256,
-    // latest execution height
-    pub execution_latest_height: u64,
+    pub l1_client_id: ClientId,
+    pub chain_id: U256,
+    pub latest_height: u64,
```

Comment:
perhaps add a comment explaining that this is the latest height of the eth execution layer?